### DISCUSSION TITLE
How to solve the material loss when using snapshot to bim file

### DISCUSSION THREAD

**Discussion by JingWang111** [iTwin/itwinjs-core#5563] [Q&A]
When I convert the dng file into the bim by using snappshot, I encounter a problem.
The dng files generated with OBD，the materials can be displayed through a custom workspace.
But when I convert the dgn file into the bim by using snapshot, the material is missing.
How to solve the problem?

![P%{C01@1R2UJSMQH0V9CZ%R](https://github.com/iTwin/itwinjs-core/assets/48142967/56b7bb83-f31a-4ce6-8ac2-881dc6bb7762)

![DAH}M){6D5$U(3O{Q{}%E89](https://github.com/iTwin/itwinjs-core/assets/48142967/8b70212d-9505-412c-a25d-ffec3d4b3420)

**Comment by arnab-git-hub** [comment]
Can you please email me the dgn and bim file to check at arnab.gogoi@bentley.com?
